src/app/components/plugin-sidebar/plugin-sidebar.component.ts

```diff
@@ -57,15 +60,22 @@ export class PluginSidebarComponent implements OnInit, OnDestroy {
     private deletedTabSubscription: Subscription | null = null;
     private changedTemplateSubscription: Subscription | null = null;
 
+    get effectiveTemplateId(): string | null {
+        if (this.useDefaultTemplate) {
```

Comment:
It seems to me that inverting this condition achieves the desired behavior.

```suggestion
        if (!this.useDefaultTemplate) {
```

Currently, selecting a plugin in the `all-plugins` template switches to the default template (`DEFAULT_UI_TEMPLATE`), and selecting a plugin in another template switches to the `all-plugins` template.

Reply:
Should be fixed now. And this time I have also tested it locally.